dubbo-rpc/dubbo-rpc-triple/src/main/java/org/apache/dubbo/rpc/protocol/tri/call/BiStreamServerCallListener.java

```diff
@@ -41,13 +49,27 @@ public void onReturn(Object value) {
 
     @Override
     public void onMessage(Object message) {
-        if (message instanceof Object[]) {
-            message = ((Object[]) message)[0];
+        if (((TripleFlowControlFrame) message).getInstance() instanceof Object[]) {
```

Comment:
the message is must be TripleFlowControlFrame？

Reply:
in my test,it must be TripleFlowControlFrame,but I added type judgment in the latest version。